recipes/gnutls/3.6.14/meta.yaml

```diff
@@ -0,0 +1,104 @@
+{% set version = "3.6.14" %}
+{% set short_version = "3.6" %}
+{% set sha256 = "5630751adec7025b8ef955af4d141d00d252a985769f51b4059e5affa3d39d63" %}
+
+package:
+  name: gnutls-pkg
+  version: "{{ version }}"
+
+about:
+  home: https://gnutls.org/
+  license: LGPL
+  summary: "The GnuTLS Transport Layer Security Library."
+
+build:
+  number: 3
```

Comment:
When we bump a version number, builds should be set to `0` (the first build of the new version)